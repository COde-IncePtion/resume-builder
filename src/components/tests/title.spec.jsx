import React from "react";
import { shallow } from "enzyme/build";
import Title from "../title";

it("should render title", () => {
  const component = shallow(<Title text="demo text" fontSize="12px" />);
  expect(component.debug()).toMatchSnapshot();
});
