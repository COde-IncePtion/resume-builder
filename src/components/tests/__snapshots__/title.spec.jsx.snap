// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render title 1`] = `
"<p style={{...}}>
  demo text
</p>"
`;
